endurain/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: endurain_app_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+
+  app:
+    image: ghcr.io/joaovitoriasilva/endurain:v0.11.2@sha256:2bd4c73e564a152ac17e336c7923e36cc6a28a3be6e67664db7ba11659455033
+    # not working rootless right now
+    # user: "1000:1000"
+    environment:
+      DB_TYPE: postgres
+      DB_HOST: endurain_db_1
+      DB_PORT: 5432
+      DB_PASSWORD: endurain
+      SECRET_KEY: ${APP_PASSWORD}
+      ENDURAIN_HOST: http://${DEVICE_DOMAIN_NAME}:8865
```

Comment:
Darn, it's a shame that we get locked in to a domain like this for the actual frontend to communicate with the backend, because it means that users trying to connect via something like tailscale or even just by local IP, or if they have multiple umbrels that are `umbrel-<#>` they won't have a working app. The only thing that will work is `http://umbrel.local`.

Not much we can do about it though.

Reply:
Yup, really a shame. I will open a ticket, maybe this can be adapted by the developer.

Reply:
I also used an environment file for this now, can you test it out?